src/components/InfoBar/infoBar.module.scss

```diff
@@ -3,7 +3,7 @@
     --bg-color: var(--grey-color-70);
     display: flex;
     border-radius: $corner-radius-xl;
-    padding: $space-m;
+    padding: $space-l-fitted;
```

Comment:
what is fitted?

Reply:
No context. we can change the name and update references.